EmployeeLeaveTracking/EmployeeLeaveTracking.Services/Services/LeaveRequestService.cs

```diff
@@ -202,10 +202,45 @@ public NewLeaveRequestDTO NewCreateNewLeaveRequest(NewLeaveRequestDTO leaveReque
                 IsViewed = false
             });
 
-
             _dbContext.Notifications.Add(notification);
             _dbContext.SaveChanges();
 
+            // Retrieve employee and manager details
+            User employee = _dbContext.Users.FirstOrDefault(u => u.Id == leaveRequest.EmployeeId);
+            User manager = _dbContext.Users.FirstOrDefault(u => u.Id == leaveRequest.ManagerId);
+
+            if (employee == null || manager == null)
+            {
+                throw new Exception("Employee or manager not found.");
```

Comment:
Message should not be hard coded. I believe we have discussed it quite a few times

Reply:
Added Message from Constant File